database/migrations/2018_06_27_201645_v2_0_0_alpha2.php

```diff
@@ -671,9 +672,9 @@ public function up()
         }
 
         // Add base payment methods
-        PaymentMethod::create(['name' => trans('fi.cash')]);
-        PaymentMethod::create(['name' => trans('fi.credit_card')]);
-        PaymentMethod::create(['name' => trans('fi.online_payment')]);
+        PaymentMethod::create(['name' => @lang('ip.cash')]);
```

Comment:
`@lang` is only available in blade templates, `trans()` would be correct to use here

Reply:
Resolved in 3aeeb66